data/decks/akuma.json

```diff
@@ -0,0 +1,165 @@
+{
+	"id": "akuma",
+	"season": 3,
+	"display_name": "Akuma",
+	"cardback": "cardback_defaultexceed.png",
+	"exceed_cost": 2,
+	"ability_effects": [
+		{
+			"timing": "set_strike",
+			"effect_type": "gauge_for_effect",
+			"per_gauge_effect": null,
+			"overall_effect": {
+				"effect_type": "critical",
+				"character_effect": true
+			},
+			"gauge_max": 1
+		},
+		{
+			"timing": "set_strike",
+			"condition": "is_critical",
+			"effect_type": "powerup_both_players",
+			"amount": 2
```

Comment:
just checking, does this work properly? wondering if there's a case where the ordering may get mixed up with the crit trigger (if so, changing this to during_strike may be smoother)

Reply:
It does work properly, I think it does not matter right now but I can change it if you want.

Reply:
set_strike should go in order I think?